crates/rover-client/src/operations/subgraph/check_workflow/runner.rs

```diff
@@ -305,6 +324,51 @@ fn get_lint_response_from_result(
     }
 }
 
+fn get_proposals_response_from_result(
+    target_url: Option<String>,
+    task_status: Option<CheckWorkflowTaskStatus>,
+    task: Option<SubgraphCheckWorkflowQueryGraphCheckWorkflowTasksOnProposalsCheckTask>,
```

Comment:
not blocking this PR but if you want this to be a tad more readable you can add a type alias in `./types.rs` for the auto-generated type like so:

```rust
pub type ProposalsCheckTask = self::subgraph_check_workflow_query::SubgraphCheckWorkflowQueryGraphCheckWorkflowTasksOnProposalsCheckTask;
```

then this argument could become

```suggestion
    task: Option<ProposalsCheckTask>,
```

Reply:
I ended up calling this ProposalsCheckTaskUnion because I panicked when there was a collision with the other ProposalsCheckTask type